src/config.js

```diff
@@ -38,7 +37,7 @@ module.exports = (store) => {
         } catch (err) {
           return callback(err)
         }
-        if (key !== undefined && !_has(config, key)) {
+        if (key !== undefined && !_get(config, key)) {
```

Comment:
If the `key` sent to `dlv` happens to not be a string, the original object might be returned. I think this is causing failures in the js-ipfs tests. We should probably convert / do some type checking.